Utilities/Doxygen/DoxygenConfig.cmake

```diff
@@ -57,6 +57,8 @@ set(DOXYGEN_FILTER_SOURCE_FILES "YES")
 set(DOXYGEN_SOURCE_BROWSER "YES")
 set(DOXYGEN_REFERENCED_BY_RELATION "YES")
 set(DOXYGEN_REFERENCES_RELATION "YES")
+set(DOXYGEN_GENERATE_TREEVIEW "YES")
+set(DOXYGEN_DISABLE_INDEX "YES")
```

Comment:
In #5180 my proposed patch suggested to use `NO` for both, but in the PR I see the choice has been made for `YES`. I think a good choice (would also  have been good to mention this with #5180)